src/ripple/resource/impl/ResourceManager.cpp

```diff
@@ -18,90 +18,101 @@
 //==============================================================================
 
 #include <BeastConfig.h>
-#include <ripple/basics/chrono.h>
 #include <ripple/resource/ResourceManager.h>
+#include <ripple/basics/chrono.h>
+#include <ripple/basics/Log.h>  // JLOG
 #include <beast/threads/Thread.h>
 #include <beast/cxx14/memory.h> // <memory>
 
 namespace ripple {
 namespace Resource {
 
-class ManagerImp
-    : public Manager
-    , public beast::Thread
+class ManagerImp : public Manager
 {
 private:
-    beast::Journal m_journal;
-    Logic m_logic;
+    beast::Journal journal_;
```

Comment:
Good point.  My inclination is to leave it (but rename it to `j_` since that is the currently approved name of member `Journal`s).  Since the `Journal` is injected it can be hard to grab one if we don't keep a copy.  Another approach would be to get the `Journal` that the `Logic` holds.  But the `Logic` doesn't (currently) have an interface that allows access to its `Journal`.